cou/cli.py

```diff
@@ -45,9 +44,13 @@ def parse_args(args: Any) -> argparse.Namespace:
     :rtype: argparse.Namespace
     """
     parser = argparse.ArgumentParser(
-        description="description",
+        description="Charmed OpenStack Upgrader(cou) is an application to upgrade Charmed "
+        "OpenStack. Application supports:\n  - Focal/Ussuri -> Focal/Victoria "
+        "\n  - Focal/Victoria -> Focal/Xena\n  - Focal/Xena -> Focal/Wallaby\n  - Focal/Wallaby "
+        "-> Focal/Yoga \nupgrades.",
```

Comment:
This is bad idea, since I can guarantee that you will forgot to update it when we support something new.

Reply:
I've moved supported releases to README.md